source/sdk/Exception.ooc

```diff
@@ -76,7 +76,7 @@ ExceptionContext: class {
 exceptionContext := ExceptionContext new()
 
 _pushStackFrame: func -> StackFrame { exceptionContext pushStackFrame() }
-_popStackFrame: func -> StackFrame { exceptionContext popStackFrame() }
+_popStackFrame: func { exceptionContext popStackFrame() }
```

Comment:
I think these functions are used only in rock, it was like that even before I worked on this.
`pop` here means simply `take first and delete it`. Internally rock uses the last pushed stack frame (`source/rock/middle/Try.ooc`), so I think `push` returns only for convenience.